WalletWasabi.Gui/ViewModels/WasabiDocumentTabViewModel.cs

```diff
@@ -71,13 +75,23 @@ public virtual void OnDeselected()
 			IsSelected = false;
 		}
 
-		public virtual void OnOpen()
+		void IDockableViewModel.OnOpen()
```

Comment:
The Docking System is what calls OnOpen, because we implement `OnOpen`, however we no longer want that to be the method we get people to override.

This is whats called an explicit interface implementation. Because its implemented explicitly it means that the interface is happy, when our object is casted to `IDockableViewModel` however people who are using our class `WasabiDocumentTabViewModel` will not see the `OnOpen(void)` method, they will only see the `OnOpen(CompositeDisposable)` method, essentially we are using it to hide (kind of make private) `OnOpen(void)` so that the users of this class dont get confused with `OnOpen(CompositeDisposable)`. To see the `OnOpen(void)` you need to cast to `IDockableViewModel`.

Reply:
Added comment to clarify as discussed.